app/src/main/kotlin/de/janhopp/luebeckmensawidget/WidgetConfigurationActivity.kt

```diff
@@ -0,0 +1,46 @@
+package de.janhopp.luebeckmensawidget
+
+import android.appwidget.AppWidgetManager.EXTRA_APPWIDGET_ID
+import android.appwidget.AppWidgetManager.INVALID_APPWIDGET_ID
+import android.content.Intent
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.activity.enableEdgeToEdge
+import androidx.glance.appwidget.updateAll
+import androidx.lifecycle.lifecycleScope
+import de.janhopp.luebeckmensawidget.ui.config.WidgetConfigurationScreen
+import de.janhopp.luebeckmensawidget.ui.theme.MensaTheme
+import de.janhopp.luebeckmensawidget.widget.MensaWidget
+import kotlinx.coroutines.launch
+
+class WidgetConfigurationActivity : ComponentActivity() {
+    private var appWidgetId = INVALID_APPWIDGET_ID
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        enableEdgeToEdge()
+
+        appWidgetId = intent?.extras?.getInt(EXTRA_APPWIDGET_ID, INVALID_APPWIDGET_ID)
+            ?: INVALID_APPWIDGET_ID
+
+        setContent {
+            MensaTheme {
+                WidgetConfigurationScreen(
+                    onConfigurationFinished = {
+                        lifecycleScope.launch {
+                            MensaWidget().updateAll(applicationContext)
```

Comment:
That doesn't work on my machine 😱 In the logs it says that the widget is updated, but the configuration changes are not applied.

Reply:
Let's make it an issue later, if it persists?